src/renderer/components/Login-Styles.js

```diff
@@ -4,9 +4,30 @@ import {
   Button,
   InputGroup,
   FormGroup,
-  Intent
+  Intent,
+  ProgressBar
 } from '@blueprintjs/core'
 
+export const DeltaBlueButton = styled.div`
+  -webkit-appearance: button-bevel;
+  background-color: transparent;
+  color: ${props => props.theme.deltaFocusBlue};;
```

Comment:
after #911 the variables will have different names so we need to remember to change this line when merging #911

Reply:
good to know, lets add this as an todo to #911